gallery_dl/formatter.py

```diff
@@ -60,14 +60,21 @@ class StringFormatter():
     - "u": calls str.upper
     - "c": calls str.capitalize
     - "C": calls string.capwords
-    - "j". calls json.dumps
+    - "g": calls text.slugify()
+    - "j": calls json.dumps
     - "t": calls str.strip
+    - "T": calls util.datetime_to_timestamp_string()
     - "d": calls text.parse_timestamp
-    - "U": calls urllib.parse.unescape
+    - "s": calls str()
     - "S": calls util.to_string()
-    - "T": calls util.to_timestamü()
+    - "U": calls urllib.parse.unescape
+    - "r": calls repr()
+    - "a": calls ascii()
```

Comment:
Would be nice if these were in ABC order.

Reply:
That was the order in https://github.com/mikf/gallery-dl/blob/master/docs/formatting.md#conversions

Reply:
Time to fix it.

Reply:
conversions are ordered by functionality, or at least that's what I was trying to do.
`l`ower, `u`pper, `c`apitalize, and `C`apwords are all responsible for modifying case, so they are grouped together.

`s`, `r`, and `a` are together since they are the default conversions also available in regular format strings. `S` fits behind regular `s`, is what I thought.
(https://docs.python.org/3/library/string.html#format-string-syntax)